### PR TITLE
Create pull request template

### PR DESCRIPTION
**Pull request by y-lohse**
Here's a proposal for our PR template.

### CODE REVIEW

**Comment by CPatchane** [comment]
Great! But I think the checkbox list syntax is "* [ ]", isn't it?

**Comment by y-lohse** [comment]
They both work :)

- [ ] with a -
* [ ] with a *

**Comment by CPatchane** [comment]
Ah ok, my bad so 👍

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Before merging this PR, the following things must have been done:
+
+- [ ] Faithful integration of the mockups at all screen sizes
+- [ ] Tested on supported browsers, including responsive mode (IE11 and Edge, Chrome >= 42, 2 latest versions of Firefox and Safari)
+- [ ] Localized in english and french
+- [ ] All changes have test coverage
+- [ ] Updated README, if necessary
+
+If this PR relies on changes in one of our internal libraries (coey-client-js or coy-ui, for example), please also do the following things:
```
Comment (GoOz) [inline PULL_REQUEST_TEMPLATE.md]: s/coey/cozy
s/coy/cozy

**Pull request merged**